src/openms/source/ANALYSIS/OPENSWATH/PeakIntegrator.cpp

```diff
@@ -45,6 +45,26 @@ namespace OpenMS
 
   PeakIntegrator::~PeakIntegrator() {}
 
+  double PeakIntegrator::estimateBackground(
```

Comment:
The `integration_type_` parameter will determine how the baseline is calculated.  I could see an if/else loop where based on the value, the background type is estimated.